commit 0c6df6d17ab43cdb5c889d49149aeb9160e6cb45
Author: Vasily Gorbik <gor@linux.ibm.com>
Date:   Sun Jun 24 09:21:59 2018 +0200

    s390/dasd: correct numa_node in dasd_alloc_queue
    
    [ Upstream commit b17e3abb0af404cb62ad4ef1a5962f58b06e2b78 ]
    
    The numa_node field of the tag_set struct has to be explicitly
    initialized, otherwise it stays as 0, which is a valid numa node id and
    cause memory allocation failure if node 0 is offline.
    
    Acked-by: Stefan Haberland <sth@linux.ibm.com>
    Signed-off-by: Vasily Gorbik <gor@linux.ibm.com>
    Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/s390/block/dasd.c b/drivers/s390/block/dasd.c
index a9f60d0ee02e..7c732414367f 100644
--- a/drivers/s390/block/dasd.c
+++ b/drivers/s390/block/dasd.c
@@ -3127,6 +3127,7 @@ static int dasd_alloc_queue(struct dasd_block *block)
 	block->tag_set.nr_hw_queues = nr_hw_queues;
 	block->tag_set.queue_depth = queue_depth;
 	block->tag_set.flags = BLK_MQ_F_SHOULD_MERGE;
+	block->tag_set.numa_node = NUMA_NO_NODE;
 
 	rc = blk_mq_alloc_tag_set(&block->tag_set);
 	if (rc)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1146_0c6df6d17ab43cdb5c889d49149aeb9160e6cb45.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1146_0c6df6d17ab43cdb5c889d49149aeb9160e6cb45.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
